source/index.html.md

```diff
@@ -167,6 +169,168 @@ HttpWebResponse response = (HttpWebResponse)request.GetResponse();
 
 ```
 
+```java
+import java.io.File;
+import java.io.FileWriter;
+import kong.unirest.Unirest;
+import kong.unirest.HttpResponse;
+
+public class recognize {
+    public static void main(String[] args){
+        String token = "MY_API_KEY";
+        String file = "C:\\assets\\demo.jpg";
+        
+        try{
+            HttpResponse<String> response = Unirest.post("https://api.platerecognizer.com/v1/plate-reader/")
+            .header("Authorization", "Token "+token)
+            .field("upload", new File(file))
+            .asString();
+            System.out.println("Recognize:");
+            System.out.println(response.getBody().toString());
+        }
+        catch(Exception e){
+            System.out.println(e);
+        }
```

Comment:
Just include the select lines.
Add a comment with a link to the full source code.

Reply:
btw that thing with the ��try{ does not show up in the code editor